### ISSUE TITLE
can I have "Ctrl+c+c" or "Ctrl+C+C" like key combination?

### ISSUE THREAD

**Issue by conanchen** [boppreh/keyboard#261]
can I have "Ctrl+c+c" or "Ctrl+C+C" like key combination?

I want to copy&paste the selected text into my application and then open it immediately using key combination:e.g. "Ctrl+C+C".

**Comment by Futurama39** [comment]
I don't believe that you can press C while pressing C. keyboard can send a signal for pressing down C, but if it sends another, that will do nothing since C is already pressed.
